xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -384,7 +392,7 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value that indicates whether the handler sends an Authorization header with the request.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the handler sends an Authorization header with the request; otherwise, <see langword="false" />.</value>
```